mipengine/controller/api/algorithm_specifications_dtos.py

```diff
@@ -34,8 +34,14 @@ class ParameterType(str, Enum):
     INT = "int"
     TEXT = "text"
     BOOLEAN = "boolean"
-    ENUM_FROM_LIST = "enum_from_list"
-    ENUM_FROM_CDE = "enum_from_cde"
+
+
+@unique
+class ParameterEnumType(str, Enum):
+    LIST = "list"
+    INPUTDATA_CDE_ENUMS = "inputdata_CDE_enums"
+    CDE_ENUMS = "CDE_enums"
+    INPUTDATA_CDES = "inputdata_CDEs"
```

Comment:
When I saw the ticket I thought I understood but now I am confused. The names `inputdata_cde_enums`, `cde_enums` and `inputdata_cdes` are very confusing. 

I think I understand the difference of `inputdata_cde_enums` and `cde_enums` (enums coming from input variable VS enums coming from fixed variable, right?). 

But what is `inputdata_cdes`? To me this means exactly the same thing as `inputdata_cde_enums`. The only difference in the names is the word "enums" but we are talking about enums in both cases, right?

Reply:
After a discussion it was decided to go with the following enum types:
`list`, `input_var_cde_enums`, `fixed_var_cde_enums`, `input_var_names`